scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -1255,12 +1265,18 @@ cdef class NumericalInversePolynomial(Method):
     Parameters
     ----------
     dist : object
-        An instance of a class with a ``pdf`` and optionally a ``cdf`` method.
+        An instance of a class with a ``pdf`` or ``logpdf`` method,
+        optionally a ``cdf`` method.
 
         * ``pdf``: PDF of the distribution. The signature of the PDF is expected to be:
           ``def pdf(self, x: float) -> float``, i.e., the PDF should accept a Python
           float and return a Python float. It doesn't need to integrate to 1,
-          i.e., the PDF doesn't need to be normalized.
+          i.e., the PDF doesn't need to be normalized. This method is optional,
+          but either ``pdf`` or ``logpdf`` need to be specified.
+        * ``logpdf``: The log of the PDF of the distribution. The signature is
+          the same as for ``pdf``. Similarly, normalization constants of the
+          PDF can be ignored. This method is optional, but either ``pdf`` or
```

Comment:
In the log-space, the normalization constants are additive rather than multiplicative, right? If this is implied, no problem.

Reply:
I think it's implied. But +1 for explicitly stating it for clarity.

Reply:
I think this is implied since we say "normalization constants of the pdf", so if we take the log, it is just the log of that constant. I can make it more clear if we decide to change something else, otherwise I would rather save the CI resouces :)

Reply:
Changed to "Similarly, log of the normalization constant of the PDF can be ignored." (my bad, I forgot "the"...)